packages/babel-helpers/src/helpers.js

```diff
@@ -23,9 +19,9 @@ helpers.typeof = defineHelper(`
 
     return _typeof(obj);
   }
-`);
+`;
 
-helpers.jsx = defineHelper(`
+helpers.jsx = () => template.program.ast`
```

Comment:
This just avoids parsing the helper snippets until the first time they are used.